Assets/General/Scripts/Console/FreeLookCam/FreeLookCamController.cs

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+// Name: FreeLookCamController.CS
+// Author: Nabil Tagba
+// Overview: Handles movement for the free look cam
+*****************************************************************************/
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// Free look cam movement
+/// </summary>
+public class FreeLookCamController : MonoBehaviour
+{
+    private PlayerInputMap _playerInput;
+
+    [Header("Settings")]
+    [SerializeField] private float _speed;
```

Comment:
Probably should specify this as movement speed. The variables in this script could be more specific in name

Reply:
^